protocols/meson.build

```diff
@@ -47,6 +47,8 @@ foreach p : protocols
 	wl_protos_headers += custom_target(
 		xml.underscorify() + '_server_h',
 		input: xml,
+		install: true,
+		install_dir: join_paths(get_option('includedir'), 'hyprland/protocols'),
```

Comment:
This will install, to e.g.`-I${includedir}/hyprland` -- as provided in the pkg-config file -- a header named `protocols/@BASENAME@-protocol.h`. For example, you'd then use:
```
#include <protocols/idle-protocol.h>
```

To include the file `/usr/include/hyprland/protocols/idle-protocol.h`

Can you confirm that's what you want? Because it's not what the Makefile does in this PR.

Reply:
If so then you don't want to put `subdirs: 'hyprland',` into the pkg-config file. That adds `/path/to/nix-store/include/hyprland` to the include path but if "hyprland/" is part of the `#include` name then you need `/path/to/nix-store/include` on the include path.